packages/lib/src/__tests__/acting-io.test.ts

```diff
@@ -197,11 +201,11 @@ describe('acting-io', () => {
 			expect(getIdFromUser).toHaveBeenCalledTimes(0)
 
 			expect(formatAndWriteItemSpy).toHaveBeenCalledTimes(1)
-			expect(formatAndWriteItemSpy).toHaveBeenCalledWith(command, resultItem)
+			expect(formatAndWriteItemSpy).toHaveBeenCalledWith(commandAndConfig, commandAndConfig, resultItem)
```

Comment:
I don't understand why some of these would be passed twice like this.

Reply:
It's an artifact of the fact that these deprecated commands take the command and the config in as the same argument and then use them for both of these arguments when they call things like `formatAndWrite`. Once we fix everything to use the new way, this will all go away.

Reply:
I wondered if I shouldn't fix them all in this pull request but it was getting rather large already.